app/scripts/search/search.js

```diff
@@ -20,8 +20,31 @@
 			};
 		})
     .controller( 'searchCtrl', ['$scope', '$http', function( $scope, $http ) {
-        $http.get("scripts/location.json").success(function(data) {
-          $scope.locations = data.venues;
-        });
+      console.log($scope, 'directly in controller')
+      var p1 = new Promise(function(resolve, reject) {
+        var attemptsLeft = 20;
+        function check() {
+          if(window.artHop && window.artHop.venues.length !== 0) {
+            resolve(window.artHop.venues);
+          } else {
+            console.log('LOOPING')
```

Comment:
I am not sure what is angulars best practices with console logs. @rxsharp Any advice here?